src/card_operations/card_pair.c

```diff
@@ -163,6 +167,50 @@ static void init_and_pair_card(card_operation_data_t *card_data,
 /*****************************************************************************
  * STATIC FUNCTIONS
  *****************************************************************************/
+
+static uint8_t atecc_nfc_sign_hash(const uint8_t *hash, uint8_t *sign) {
+  atecc_data.retries = DEFAULT_ATECC_RETRIES;
+
+  bool usb_irq_enable_on_entry = NVIC_GetEnableIRQ(OTG_FS_IRQn);
+  NVIC_DisableIRQ(OTG_FS_IRQn);
+  do {
+    if (atecc_data.status != ATCA_SUCCESS)
+      LOG_CRITICAL("PAIR SG: %04x, count:%d",
+                   atecc_data.status,
+                   DEFAULT_ATECC_RETRIES - atecc_data.retries);
+    atcab_init(atecc_data.cfg_atecc608a_iface);
+    atecc_data.status = atcab_sign(slot_3_nfc_pair_key, hash, sign);
+  } while (atecc_data.status != ATCA_SUCCESS && --atecc_data.retries);
+  if (usb_irq_enable_on_entry == true)
+    NVIC_EnableIRQ(OTG_FS_IRQn);
+
+  return atecc_data.status;
+}
+
+static uint8_t atecc_nfc_ecdh(const uint8_t *pub_key, uint8_t *shared_secret) {
```

Comment:
add prototypes of these function under the `"STATIC FUNCTION PROTOTYPES"` section of this file.

Reply:
ok